packages/core/src/lib/telemetry.ts

```diff
@@ -181,22 +187,24 @@ export function printTelemetryStatus () {
   }
 
   console.log()
-  console.log(`Telemetry will be sent the next time you run ${g`"keystone dev"`}, unless you opt-out`)
+  printNext(telemetry)
 }
 
-function inform () {
-  const { telemetry, userConfig } = getDefaultedTelemetryConfig()
-
-  // no telemetry? somehow we missed something, do nothing
-  if (!telemetry) return
-
+function inform (
+  telemetry: TelemetryOK,
+  userConfig: Configuration
+) {
   console.log() // gap to help visiblity
   console.log(`${bold('Keystone Telemetry')}`)
   printAbout()
   console.log(`You can use ${g`"keystone telemetry --help"`} to update your preferences at any time`)
   console.log()
-  console.log(`No telemetry data has been sent, but telemetry will be sent the next time you run ${g`"keystone dev"`}, unless you opt-out`)
+  if (telemetry.informedAt === null) {
+    console.log(`No telemetry data has been sent as part of this notice`)
+  }
+  printNext(telemetry)
   console.log() // gap to help visiblity
+  console.log(`For more information, including how to opt-out see ${grey`https://keystonejs.com/telemetry`} (updated ${b`2024-08-15`})`)
```

Comment:
As everyone will be re-informed, this date helps to show that the information has been updated (however small)